File: youqugou-web-manager/src/main/webapp/js/service/sellerService.js
app.service('sellerService',function($http){
	this.searchSeller = function (currentPage, itemsPerPage, searchEntity) {
		return $http.post("../seller/searchSeller.do?pageNum=" + currentPage
			+ "&pageSize=" + itemsPerPage, searchEntity);
    }

    this.selectSellerById = function (sellerId) {
		return $http.get("../seller/selectSellerById.do?sellerId=" +sellerId);
    }
});

File: youqugou-interface/src/main/java/cn/itcast/core/service/SellerService.java
package cn.itcast.core.service;

import cn.itcast.core.entity.PageResult;
import cn.itcast.core.pojo.seller.Seller;

public interface SellerService {
    void insertSeller(Seller seller);

    PageResult searchSeller(Integer pageNum, Integer pageSize, Seller seller);

    Seller selectSellerById(String sellerId);
}
